### PR TITLE
Add git-branch-new icon

### PR DESCRIPTION
**Pull request by mittalyashu**
| Preview |
| --- |
| CleanShot 2021-01-27 at 21 22 36@2x |

### CODE REVIEW

**Comment by ericfennis** [comment]
Can you rename this to: `git-branch-plus` for more constancy?

**Comment by mittalyashu** [comment]
Done

**Comment by ericfennis** [comment]
nice!

**Review by ericfennis — APPROVED** [review]

File: icons/git-branch-plus.svg [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M6 3V15" />
+  <path d="M18 9C19.6569 9 21 7.65685 21 6C21 4.34315 19.6569 3 18 3C16.3431 3 15 4.34315 15 6C15 7.65685 16.3431 9 18 9Z" />
+  <path d="M6 21C7.65685 21 9 19.6569 9 18C9 16.3431 7.65685 15 6 15C4.34315 15 3 16.3431 3 18C3 19.6569 4.34315 21 6 21Z" />
+  <path d="M15 6C12.6131 6 10.3239 6.94821 8.63604 8.63604C6.94821 10.3239 6 12.6131 6 15" />
+  <path d="M18 15V21" />
+  <path d="M21 18H15" />
```
Comment (ericfennis) [inline icons/git-branch-plus.svg:17]: Svgomg had some code improves for use.

```suggestion
  <path d="M6 3v12" />
  <path d="M18 9a3 3 0 100-6 3 3 0 000 6z" />
  <path d="M6 21a3 3 0 100-6 3 3 0 000 6z" />
  <path d="M15 6a9 9 0 00-9 9" />
  <path d="M18 15v6" />
  <path d="M21 18h-6" />
```

**Pull request merged**